ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -3961,6 +3961,51 @@ interface Ethernet81/2
 interface Ethernet81/10
    description isis_port_channel_member
    channel-group 110 mode active
+!
+interface Ethernet82
+   description Switchport_tap_tool
+   switchport tap native vlan 10
+   switchport tap identity 3 inner 5
+   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
+   switchport tap mpls pop all
+   switchport tap encapsulation gre protocol 0x0001 strip
+   switchport tap encapsulation gre protocol 0x0002 feature header length 3 strip
+   switchport tap encapsulation gre protocol 0x0003 feature header length 2 strip re-encapsulation ethernet
+   switchport tap encapsulation gre protocol 0x0004 strip re-encapsulation ethernet
+   switchport tap encapsulation gre strip
```

Comment:
this line disappears in my config

Reply:
its there for me -

```
interface Ethernet10
   description Switchport_tap_tool
   switchport tap native vlan 10
   switchport tap identity 3 inner 5
   switchport tap mac-address dest 01:00:00:00:00:00 src 01:23:45:67:89:ab
   switchport tap encapsulation gre protocol 0x1 strip
   switchport tap encapsulation gre protocol 0x2 feature header length 3 strip
   switchport tap encapsulation gre protocol 0x3 feature header length 2 strip re-encapsulation ethernet
   switchport tap encapsulation gre protocol 0x4 strip re-encapsulation ethernet
   switchport tap encapsulation gre strip
   switchport tap encapsulation gre destination 1.1.1.1 source 1.1.1.2 protocol 0x0 strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x10 strip
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x11 feature header length 2 strip re-encapsulation ethernet
   switchport tap encapsulation gre destination 2.1.1.2 protocol 0x12 strip re-encapsulation ethernet
   switchport tap mpls pop all
   switchport tool mpls pop all
   switchport tool encapsulation vn-tag strip
   switchport tool encapsulation dot1br strip
   switchport tap allowed vlan 25
   switchport tool allowed vlan 23
   switchport tool identity qinq
   switchport tool identity dot1q source dzgre port
   switchport tap truncation 150
   switchport tool truncation 1
   switchport tap default group g1 group g2 group g3
   switchport tap default nexthop-group nexthop_g1 nexthop_g2 nexthop_g3
   switchport tap default interface Ethernet4
   switchport tap default interface Po10
   switchport tool group set group1 group2
   switchport tool dot1q remove outer 1
```

Reply:
tested multiple ways, found out that -  it removes that command when we apply `switchport tap encapsulation gre destination 2.1.1.2 strip` i.e. without specifying protocol and vice-versa.

Reply:
Can we add this in the description?

Reply:
Added!